src/logging.ts

```diff
@@ -1,8 +1,8 @@
 'use strict';
 
 export enum LogLevel {
-  Verbose,
   Debug,
+  Verbose,
```

Comment:
Is changing how logs are written a minor thing?

Reply:
IMO, in this case it is.  I'm not changing how logs are written per se, I'm fixing a bug in the filtering so it's more inline with what someone would expect.

Debug logging should create more "Stuff" than Verbose logging.

Reply:
I meant that you're fixing a bug, that's not a maint commit. It changes what is logged. If you had it verbose before to get all the logs (like me), this changes behavior.